contracts/SpokePool.sol

```diff
@@ -121,6 +125,8 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
         address caller
     );
     event EmergencyDeleteRootBundle(uint256 indexed rootBundleId);
+    event PausedDeposits(bool indexed isPaused);
+    event PausedFills(bool indexed isPaused);
```

Comment:
Minor: Do these need to be indexed? I imagine that there will be so few of them that there's not much advantage to filtering on the "paused" state.

Reply:
+1